install.sh

```diff
@@ -42,12 +42,20 @@ fi
 
 usageFunction()
 {
-  log "Usage: $0 (-n) (-h)" 2
-  log "\t-n Non-interactive installation (Optional)" 2
-  log "\t-h Show usage" 2
+  log "Usage: $0 (-n) (-h)" $COLOR_GREEN
+  log "\t-n Non-interactive installation (Optional)" $COLOR_GREEN
+  log "\t-h Show usage" $COLOR_GREEN
   exit 1
 }
 
+# Check if .env file exists and load vars from env file
+if [ -f .env ]; then
+    export $(grep -v '^#' .env | xargs)
+else
+    log "Error: .env file not found, copy/paste the .env-dist file to .env and edit it" $COLOR_RED
```

Comment:
Perhaps add an action that automatically copies the `.env-dist` file to `.env`, so something like

```bash
mv .env-dist .env && vim .env
```

Reply:
Yes I will add it

Reply:
But wait, we ask this to the user after, so maybe I will move this.
I will correct

Reply:
I can't do this, this section is for the non-interactive install, so I've moved the export section to the non-interactive condition.